src/Momento.Sdk/Config/Configurations.cs

```diff
@@ -107,7 +107,7 @@ public static IConfiguration V1(ILoggerFactory? loggerFactory = null)
                 IRetryStrategy retryStrategy = new FixedCountRetryStrategy(finalLoggerFactory, maxAttempts: 3);
                 ITransportStrategy transportStrategy = new StaticTransportStrategy(
                     loggerFactory: finalLoggerFactory,
-                    maxConcurrentRequests: 200,
+                    maxConcurrentRequests: 400, // max of 4 connections https://github.com/momentohq/client-sdk-dotnet/issues/460
```

Comment:
technically we have made a guarantee not to change this configuration, and we would want to consider adding a v2 here and leaving v1 intact.

In practice I think this change is safe and invisible enough to where it's not worth adding the complexity of adding a v2 for this. So I'm good with this as-is, just wanted to call it out for future consideration.